addons/pos_mercury/models/pos_mercury.py

```diff
@@ -48,8 +48,16 @@ def _compute_prefixed_card_number(self):
 class PoSPaymentMethod(models.Model):
     _inherit = 'pos.payment.method'
 
-    pos_mercury_config_id = fields.Many2one('pos_mercury.configuration', string='Mercury Credentials', help='The configuration of Mercury used for this journal')
+    pos_mercury_config_id = fields.Many2one('pos_mercury.configuration', string='Vantiv Credentials', help='The configuration of Vantiv used for this journal')
 
+    def _get_payment_terminal_selection(self):
+        return super(PoSPaymentMethod, self)._get_payment_terminal_selection() + [('mercury', 'Vantiv')]
+
+    @api.onchange('use_payment_terminal')
+    def _onchange_use_payment_terminal(self):
```

Comment:
@caburj We should probably also have this onchange on the 'adyen' module to set 'adyen_api_key' and 'adyen_terminal_identifier' to false.